src/test/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthServiceTest.java

```diff
@@ -343,12 +350,48 @@ void GIVEN_GG_with_cda_WHEN_updated_ca_type_THEN_ca_is_updated()
         assertThat(initialCA, not(thirdCA));
         assertThat(getCaPassphrase(), not(initialCaPassPhrase));
 
-        verify(client, times(3)).putCertificateAuthorities(putCARequestArgumentCaptor.capture());
+        verify(client, times(2)).putCertificateAuthorities(putCARequestArgumentCaptor.capture());
```

Comment:
This is not a mistake, before this changes we were unnecessarily configuring CAs 1 time. When no `ca_type` is provided we default to use RSA and if we then change the configuration  to RSA, the config is effectively the same. The CA types is still RSA so we don't need to reconfigure them again.